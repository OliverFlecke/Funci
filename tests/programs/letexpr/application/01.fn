main :: Int
     = let f x :: (Int -> Int) = 2 * x
       in let g x :: (Int -> Int) = -x
            in g (f 2)
