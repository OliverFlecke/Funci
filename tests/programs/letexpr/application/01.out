Number (I -4) (Unit [])